scripts/curl_install_pypi/install.py

```diff
@@ -24,6 +24,7 @@
 import shutil
 import subprocess
 import hashlib
+import distro
```

Comment:
1. Don't need to add dependency requirement and update *.py3.*.txt and setup.py?
2. According to https://pypi.org/project/distro/ , "Distro currently supports Linux and BSD based systems but Windows and OS X support is also planned."  What about Windows platform?